rcl/test/rcl/test_client.cpp

```diff
@@ -121,7 +121,7 @@ TEST_F(TestClientFixture, test_client_nominal) {
 TEST_F(TestClientFixture, test_client_init_fini) {
   rcl_ret_t ret;
   // Setup valid inputs.
-  rcl_client_t client;
+  rcl_client_t client = rcl_get_zero_initialized_client();
```

Comment:
Can you explain more about why we need this fix?  This is already done down below in line 137 (before client is ever used), so I don't see why this change is necessary.

Reply:
ah, that is a true. maybe we don't need this line at all. i was just thinking that `rcl_get_zero_initialized_client` should be used to have initialized structure just in case.